docs/kpod-save.1.md

```diff
@@ -26,6 +25,9 @@ using the **output** flag. The **quiet** flag suppresses the output when set.
 **--output, -o**
 Write to a file, default is STDOUT
 
+**--format**
+Save image to either oci-archive or docker-archive
```

Comment:
Can you include what the names of the options are here? It's not clear what I need to type to get an OCI format, for example - is it `--format oci` (you use that below) or `--format oci-archive` as this description would imply

Reply:
fixed.